### ISSUE TITLE
Plugin cleanup action items

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-mstoday#95]
- [ ] Restrict SiteOrigin from loading code on pages where its not used, especially homepage
- [ ] Same with the Inline Google Spreadsheet plugin
- [ ] Create recommendation for consolidating optimization plugins

**Comment by MirandaEcho** [comment]
@joshdarby is the last item here done as well (its probably in a separate issue that I missed)?

**Comment by joshdarby** [comment]
@MirandaEcho It has not been done yet due to a couple of reasons:
- This issue ran out of time
- I assumed we may be waiting on Lauchlin to let us know whether or not she wanted us on a call with Cloudflare to see if we could get it up and running as it should be, since we'd probably be able to remove most (if not all) of the optimization plugins once it's functioning as expected.

**Comment by MirandaEcho** [comment]
Got it, thanks! Do we have updated page speed scores after wrangling SiteOrigin and Inline Spreadsheet Viewer?

**Comment by joshdarby** [comment]
@MirandaEcho Kind of, but only from local environments. @benlk added a screenshot in https://github.com/INN/umbrella-mstoday/pull/97#issuecomment-658395065 of a before/after

Once we put this on staging we can get a better estimate of how much it helped.

**Comment by MirandaEcho** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Plugin audit & cleanup](https://trello.com/c/yJqWAQ9J/20-plugin-audit-cleanup)
